hasher-matcher-actioner/src/OpenMediaMatch/app.py

```diff
@@ -97,7 +97,7 @@ def create_app() -> flask.Flask:
     with app.app_context():
         # We only run apscheduler in the "outer" reloader process, else we'll
         # have multiple executions of the the scheduler in debug mode
```

Comment:
You may consider adding a note about gunicorn production code, and the dangers/desirability of running multiple schedulers there.